client/src/Routes.js

```diff
@@ -15,7 +15,7 @@ const Routes = () => {
       <Route path="/" exact component={Home} />
       <Route path="/about/" component={About} />
       <Route path="/status/" component={Status} />
-      <Route path="/applicant_login" component={ApplicantLogin} />
+      <Route path="/applicant_login" component={Login} />
```

Comment:
woops, we should also change this path to /login

Reply:
Done. Thanks.